lib/train/transports/local.rb

```diff
@@ -21,13 +23,18 @@ def initialize(options)
         super(options)
         @cmd_wrapper = nil
         @cmd_wrapper = CommandWrapper.load(self, options)
+        @os = os
       end
 
       def run_command(cmd)
-        cmd = @cmd_wrapper.run(cmd) unless @cmd_wrapper.nil?
-        res = Mixlib::ShellOut.new(cmd)
-        res.run_command
-        CommandResult.new(res.stdout, res.stderr, res.exitstatus)
+        if defined?(@os) && @os.windows?
+          @windows_runner ||= WindowsRunner.new
```

Comment:
should we decide for the runner during initialization? we could just can `@runner.run_command(cmd)` then here